sources/ClangSharp/Interop.Extensions/clang.ResolveLibrary.cs

```diff
@@ -54,7 +59,15 @@ private static bool TryResolveClang(Assembly assembly, DllImportSearchPath? sear
 
         private static bool TryResolveClangSharp(Assembly assembly, DllImportSearchPath? searchPath, out IntPtr nativeLibrary)
         {
-            return NativeLibrary.TryLoad("libClangSharp", assembly, searchPath, out nativeLibrary);
+            if (NativeLibrary.TryLoad("libClangSharp", assembly, searchPath, out nativeLibrary)){
+                return true;
+            }
+
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux) && NativeLibrary.TryLoad("ClangSharp", assembly, searchPath, out nativeLibrary)){
```

Comment:
`libClangSharp` is only produced with the name `libClangSharp`, could you clarify why this is needed?

Reply:
I did a test and it is not needed.